code/src/main/resources/lib/access.es6

```diff
@@ -0,0 +1,19 @@
+const libs = {
+  authLib: require("/lib/xp/auth"),
```

Comment:
Just a small remark, but could this perhaps be renamed to just "auth" to keep the naming consistent with the other libs? 🥺

Reply:
done